docs/content/direct/core-chart-argocd.md

```diff
@@ -1,11 +1,21 @@
 # Using Argo CD with KubeStellar Core chart
 
+## Table of Contents
+- [Overview](#overview)
+- [Pre-requisites](#pre-requisites)
+- [Installing Argo CD using KubeStellar Core chart](#installing-argo-cd-using-kubestellar-core-chart)
+- [Deploying Argo CD applications](#deploying-argo-cd-applications)
+
+## Overview
+
 This documents explains how to use the KubeStellar core Helm chart to:
 
 - deploy Argo CD in KubeFlex hosting cluster;
 - register every WDS as a target cluster in Argo CD; and
 - create Argo CD applications as specified by the chart values.
 
+For a detailed step-by-step installation guide with expected outputs, see [Step-by-Step Installation Guide](core-chart.md).

```

Comment:
I am going hold my comments on this for now, because #2938 is in concurrent progress.

Reply:
Now that #2938 has merged there is something definite to compare with here.

The text "Step-by-Step Installation Guide" does not match the relative URL (core-chart.md). The latter refers to the whole document, not just the step-by-step part.